tools/src/test/java/org/apache/kafka/tools/ResetIntegrationTest.java

```diff
@@ -256,7 +256,6 @@ public void testResetWhenLongSessionTimeoutConfiguredWithForceOption(final TestI
         streams = new KafkaStreams(setupTopologyWithoutIntermediateUserTopic(), streamsConfig);
         streams.cleanUp();
 
-        // Reset would fail since long session timeout has been configured
```

Comment:
why remove this comment?

Reply:
It was unintentionally deleted when I was tweaking this test.

Reply:
it happens 😄